### PR TITLE
3604 edit/update button

### PR DESCRIPTION
**Pull request by maksvet**
- Button for updating pet added.
- Minor edits for routes.

### CODE REVIEW

**Comment by emmafoci** [comment]
Why does this have every single commit since the beginning of last sprint?

**Comment by maksvet** [comment]
Emma, I was trying to PR into main by mistake.

**Comment by emmafoci** [comment]
Sure no worries, just reviewing code

File: src/orbital-demo/src/app/app.component.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export class AppComponent implements OnInit, OnDestroy {
     this.response = null;
     this.request = {
       verbType: 'GET',
-      path: '/pet/1',
```
Comment (ReubenKogie) [inline src/orbital-demo/src/app/app.component.ts:63]: good eye!

**Review by ReubenKogie — APPROVED** [review]
Looks good mate!

**Pull request merged**